ci-operator/config/openshift-knative/kn-plugin-func/openshift-knative-kn-plugin-func-release-next.yaml

```diff
@@ -2,14 +2,14 @@ build_root:
   image_stream_tag:
     name: builder
     namespace: ocp
-    tag: rhel-8-golang-1.16-openshift-4.9
+    tag: rhel-8-golang-1.17-openshift-4.11
```

Comment:
I'd go straight to Go 1.18 probably to skip another inevitable update PR soon.

Reply:
Right I see, but I'm going with 1.17 to make it sync with what we have on upstream